mod/dbs.js

```diff
@@ -9,27 +9,29 @@ module.exports = () => {
   Object.keys(process.env)
     .filter((key) => key.split('_')[0] === 'DBS')
     .filter((key) => !dbs[key.split('_')[1]])
-    .forEach((key) => {
+    .forEach(async (key) => {
       if (process.env[key].match(/^postgres/))
-        return postgres(key.split('_')[1], process.env[key]);
+        return await postgres(key.split('_')[1], process.env[key]);
 
       if (process.env[key].match(/^aurora/))
         return aurora(key.split('_')[1], process.env[key]);
     });
 
-  async function postgres(key, connectionString) {
-    let secretClient = new SecretsManager({region: process.env.AWS_DEFAULT_REGION});
-    let secretId = `${process.env.XYZ_ENV}/xyz/pg`;
-    const pgSecret = await secretClient.getSecretValue({SecretId: secretId}).promise().then((data) => {
+  async function postgres(key, host) {
+    const secretClient = new SecretsManager({region: process.env.AWS_DEFAULT_REGION});
+    const secretId = `${process.env.XYZ_ENV}/xyz4/pg`;
+    const pgSecret = await secretClient.getSecretValue({SecretId: secretId}).then((data) => {
       return JSON.parse(data.SecretString);
     })
 
     // Create connection pool.
     const pool = new Pool({
-      user: pgSecret["username"],
-      password: pgSecret["password"],
-      connectionString: connectionString,
-      statement_timeout: parseInt(process.env.STATEMENT_TIMEOUT) || 10000,
+      database: 'map',
+      user: pgSecret['username'],
+      password: pgSecret['password'],
+      host,
+      port: 5432,
```

Comment:
This is less flexible as it only allows to choose the host, and fixes the port, db, and any other param that we might want to change through the envvar. But it's just an observation.

Reply:
I'm fixing that right now. We are going to use `host` when in the Pod and `connectionString` when local.

Reply:
I'm afraid that would be an anti-pattern. Just use one envvar for all envs and it should work.